src/matches.rs

```diff
@@ -328,10 +328,10 @@ fn flatten_arm_body<'a>(
                     (true, body)
                 }
             } else {
-                let cond_becomes_muti_line = opt_shape
+                let cond_becomes_multi_line = opt_shape
```

Comment:
elsewhere, `multiline` is used a lot. I'm generally performing context-free or line local corrections, and I try to make the simplest correction. If `multiline` is preferable, I'm happy to apply that change as well.